### PR TITLE
Add gvisor ci test for ngo

### PR DESCRIPTION
**Pull request by daan3601**

### CODE REVIEW

File: .github/workflows/demo_test.yml [hunk 1]
```diff
@@ -621,3 +621,22 @@ jobs:
 
     - name: Run unix socket test
       run: docker exec cross_world_unix_socket bash -c "cd /root/occlum/demos/cross_world_uds; SGX_MODE=SIM ./run_cross_world_uds_test.sh"
+
+  Gvisor_syscalls_test:
+    runs-on: ubuntu-18.04
```
Comment (guzongmin) [inline .github/workflows/demo_test.yml:626]: Please update the version to ubuntu-20.04

Comment (daan3601) [reply .github/workflows/demo_test.yml]: Modification has been completed

**Review by jessehui — APPROVED** [review]
LGTM

**Review by guzongmin — APPROVED** [review]
LGTM

**Pull request merged**